connect/runtime/src/test/java/org/apache/kafka/connect/runtime/errors/RetryWithToleranceOperatorTest.java

```diff
@@ -243,56 +241,54 @@ public void testBackoffLimit() {
         retryWithToleranceOperator.metrics(errorHandlingMetrics);
         retryWithToleranceOperator.markAsFailed();
         assertFalse("should not tolerate any errors", retryWithToleranceOperator.withinToleranceLimits());
 
-        retryWithToleranceOperator = new RetryWithToleranceOperator();
-        retryWithToleranceOperator.configure(config(RetryWithToleranceOperator.TOLERANCE_LIMIT, "all"));
+        retryWithToleranceOperator = new RetryWithToleranceOperator(ERRORS_RETRY_TIMEOUT_DEFAULT, ERRORS_RETRY_MAX_DELAY_DEFAULT, ALL, SYSTEM);
         retryWithToleranceOperator.metrics(errorHandlingMetrics);
         retryWithToleranceOperator.markAsFailed();
         retryWithToleranceOperator.markAsFailed();
         assertTrue("should tolerate all errors", retryWithToleranceOperator.withinToleranceLimits());
 
-        retryWithToleranceOperator = new RetryWithToleranceOperator();
-        retryWithToleranceOperator.configure(config(RetryWithToleranceOperator.TOLERANCE_LIMIT, "none"));
+        retryWithToleranceOperator = new RetryWithToleranceOperator(ERRORS_RETRY_TIMEOUT_DEFAULT, ERRORS_RETRY_MAX_DELAY_DEFAULT, NONE, SYSTEM);
         assertTrue("no tolerance is within limits if no failures", retryWithToleranceOperator.withinToleranceLimits());
     }
 
     @Test
     public void testDefaultConfigs() {
-        RetryWithToleranceOperatorConfig configuration;
-        configuration = new RetryWithToleranceOperatorConfig(new HashMap<>());
-        assertEquals(configuration.retryTimeout(), 0);
-        assertEquals(configuration.retryDelayMax(), 60000);
-        assertEquals(configuration.toleranceLimit(), ToleranceType.NONE);
+        ConnectorConfig configuration = config(emptyMap());
+        assertEquals(configuration.errorRetryTimeout(), ERRORS_RETRY_TIMEOUT_DEFAULT);
+        assertEquals(configuration.errorMaxDelayInMillis(), ERRORS_RETRY_MAX_DELAY_DEFAULT);
+        assertEquals(configuration.errorToleranceType(), ToleranceType.valueOf(ERRORS_TOLERANCE_DEFAULT.toUpperCase(Locale.ROOT)));
 
         PowerMock.verifyAll();
     }
 
+    ConnectorConfig config(Map<String, String> connProps) {
+        Map<String, String> props = new HashMap<>();
+        props.put(ConnectorConfig.NAME_CONFIG, "test");
+        props.put(ConnectorConfig.CONNECTOR_CLASS_CONFIG, SinkTask.class.getName());
+        props.putAll(connProps);
+        return new ConnectorConfig(plugins, props);
+    }
+
     @Test
-    public void testConfigs() {
-        RetryWithToleranceOperatorConfig configuration;
-        configuration = new RetryWithToleranceOperatorConfig(config("retry.timeout", "100"));
-        assertEquals(configuration.retryTimeout(), 100);
+    public void testSetConfigs() {
+        ConnectorConfig configuration;
+        configuration = config(singletonMap("errors.retry.timeout", "100"));
```

Comment:
Are we purposefully avoiding the use of the config constants, for example to check for changes in the constants?